pages/create.js

```diff
@@ -99,8 +101,12 @@ const StyledSuccessMessage = styled.h5`
   border-radius: 8px;
 `;
 
-//----------------------------------------------- FUNCTION------------HERE
+const StyledLinkCreate = styled(Link)`
```

Comment:
```suggestion
const StyledBackLink = styled(Link)`
```

Reply:
I would suggest you to rename your styled component to something like this, since the button is not for creating something, but to going one page back, or?

Reply:
i will do it, thanks